omero/users/cli/chgrp.rst

```diff
@@ -17,14 +17,14 @@ Who may move data
 * a full administrator
 * a :doc:`restricted administrator
   </sysadmins/restricted-admins>` with `Chgrp` privilege
-* the owner of the data *if* they are a member of the target group
+* the owner of the data *if* he/she is a member of the target group
```

Comment:
Don't see that this is needed -- singular "they" has a fine history -- but no strong objection.